input-cache/txcache/version.ctl

```diff
@@ -1 +1 @@
-1.9.369
\ No newline at end of file
+1.9.372
```

Comment:
Are the files under input-cache supposed to be checked in?

Reply:
No. I removed the input-cache folder now.